components/brave_vpn/browser/brave_vpn_service.cc

```diff
@@ -619,6 +646,7 @@ void BraveVpnService::OnCredentialSummary(const std::string& domain,
     VLOG(1) << __func__ << " : Treat it as not purchased state in android.";
     SetPurchasedState(env, PurchasedState::NOT_PURCHASED);
 #else
+    out_of_credentials_ = true;
```

Comment:
I think we don't need to have this as a member.
Just passing to `UpdatePurchasedStateForSessionExpired()` would work?

Reply:
Fixed!